cg/store/filters/status_order_filters.py

```diff
@@ -4,11 +4,14 @@
 from sqlalchemy import asc, desc, or_
 from sqlalchemy.orm import Query
 
+from cg.constants import Workflow
 from cg.server.dto.orders.orders_request import OrderSortField, SortOrder
 from cg.store.models import Customer, Order
 
 
 def filter_orders_by_workflow(orders: Query, workflow: str | None, **kwargs) -> Query:
+    if workflow == Workflow.BALSAMIC:
+        return orders.filter(Order.workflow.startswith(workflow))
```

Comment:
Should this be fixed here? Or should the call to the apply_order_filter be modified? Will we ever need a stricter balsamic filter which excludes umi and qc?

Reply:
No, this is the right spot 👍